jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -35,9 +35,11 @@
 import java.security.PrivilegedAction;
 import java.util.Objects;
 import java.util.Properties;
-/*[IF Sidecar19-SE-B165]*/
+/*[IF Sidecar19-SE]*/
```

Comment:
This change from Sidecar19-SE-B165 to Sidecar19-SE has likely broken the IBM internal b148 builds. Please check and update if necessary.